### PR TITLE
Update kubernetes-agent-tentacle to 8.1.2099

### PR DESCRIPTION
**Pull request by APErebus**
Includes two Tentacle changes:

- https://github.com/OctopusDeploy/OctopusTentacle/pull/992
- https://github.com/OctopusDeploy/OctopusTentacle/pull/994

### CODE REVIEW

**Review by kevjt — APPROVED** [review]
lgtm

File: charts/kubernetes-agent/CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ### Patch Changes
 
-- c8a7da3: Update kubernetes-agent-tentacle to 8.1.2049. Removes the `KubernetesScriptServiceV1Alpha` service
+- c8a7da3: Update kubernetes-agent-tentacle to 8.1.2099. Removes the `KubernetesScriptServiceV1Alpha` service
```
Comment (kevjt) [inline charts/kubernetes-agent/CHANGELOG.md:7]: Oops, missed this. This shouldn't change

Comment (APErebus) [reply charts/kubernetes-agent/CHANGELOG.md:7]: ah yep

Comment (APErebus) [reply charts/kubernetes-agent/CHANGELOG.md]: Fixed

**Pull request merged**